ui/app/templates/vault/cluster/access.hbs

```diff
@@ -11,6 +11,13 @@
         </LinkTo>
       </li>
     {{/if}}
+    {{#if (has-permission "access" routeParams="mfa")}}
+      <li>
+        <LinkTo @route="vault.cluster.access.mfa" data-test-link={{true}}>
```

Comment:
maybe a more specific data-test something to indicate it's for MFA?  When debugging tests, or expanding on tests I'm often searching the code for data-test-xx and the more specific the data-test is the easier it is for me to narrow down the specific one I'm looking for.

Reply:
Good point. I will revisit this as part of tests. Currently tests are using this nav collection based on data-test-link attr.